### PR TITLE
[example][doc] Add trace LLM example & little refine tracing doc

### PR DESCRIPTION
**Pull request by zhengfeiwang**
# Description

- Add a trace LLM example
- Little refine tracing doc structure
- Fix a small bug in build doc CI powershell script

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/tutorials/tracing/llm/trace-llm.ipynb [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  }
+ ],
+ "metadata": {
+  "build_doc": {
+   "author": [
+    "zhengfeiwang@github.com"
+   ],
+   "category": "local",
+   "section": "Tracing",
+   "weight": 10
```
Comment (wangchao1230) [inline examples/tutorials/tracing/llm/trace-llm.ipynb:170]: make sure the ordering is correct, this should be the first one

File: examples/tutorials/tracing/llm/trace-llm.ipynb [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    "\n",
+    "# start a trace session, and print a url for user to check trace\n",
+    "start_trace(collection=\"trace-llm\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run the LLM application again, and you should be able to see new trace logged in the trace UI."
```
Comment (wangchao1230) [inline examples/tutorials/tracing/llm/trace-llm.ipynb:131]: may add a screenshot here

File: examples/tutorials/tracing/llm/trace-llm.ipynb (hunk 1 above)
Comment (zhengfeiwang) [reply examples/tutorials/tracing/llm/trace-llm.ipynb:170]: yes, i have consulted Brynn on this, 10 should be correct value here.

File: examples/tutorials/tracing/llm/trace-llm.ipynb (hunk 2 above)
Comment (zhengfeiwang) [reply examples/tutorials/tracing/llm/trace-llm.ipynb]: added

**Review by wangchao1230 — APPROVED** [review]

**Review by brynn-code — APPROVED** [review]

**Pull request merged**